src/components/cc-logs-instances/cc-logs-instances.stories.js

```diff
@@ -0,0 +1,324 @@
+import './cc-logs-instances.js';
+import { makeStory } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+/**
+ * @typedef {import('./cc-logs-instances.types.d.ts').Instance} Instance
+ * @typedef {import('./cc-logs-instances.types.d.ts').Deployment} Deployment
+ */
+
+function randomCommit () {
```

Comment:
question: `cc-header-app` uses a hard coded String in its story file, maybe we could put this a lib?